### DISCUSSION TITLE
Addition of "Order total line items" to Taxable Subject

### DISCUSSION THREAD

**Discussion by fabianhijlkema** [craftcms/commerce#2317] [Ideas]
### Description
If I'm correct there is no option to add taxes on all line items combined(!) with the exception of shipping costs. I feel this would be a convenient addition to the current options so the tax adjustments can become part of the orderAdjustments object, and not only part of the adjustments object per line item.

<img width="565" alt="screen shot 2018-04-19 at 16 36 45" src="https://user-images.githubusercontent.com/2919995/38998700-6bcad3aa-43f0-11e8-8068-390e6cfa6ad4.png">

**Comment by lukeholder** [comment]
Thanks for the feedback, it is a good idea, will do.

**Comment by robmcfadden** [comment]
We're upgrading a site to Commerce 2 and it's a bit confusing because where "Line item price" is set as the Taxable Subject in Commerce 1, they are combined (which is what we want) into one tax adjustment on the adjustments output. In Commerce 2, there's a separate tax adjustment added for each line item. It doesn't appear that we were doing something special in Commerce 1 to achieve this.

**Comment by elivz** [comment]
Just wanted to add my +1 to this. A single tax adjustment applying to the order subtotal + any discounts (but excluding shipping) would have make things much easier on a couple of projects.
